setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="staking_deposit",
-    version='2.5.6',
+    version=__version__,
```

Comment:
The import of `__version__` also seems to have gotten removed.

Reply:
Solved, added: `from staking_deposit import __version__`